_posts/2020-07-31-kubeflow-1.1-blog-post.md

```diff
@@ -0,0 +1,39 @@
+
+The ML productivity enhancements in 1.1 include end-to-end workflows using Fairing and Kale. The Fairing workflow enables users to build, train and deploy models from a notebook and [Fairing improvements](https://github.com/kubeflow/fairing/releases/tag/v1.0.1) include the support of configuring environment variables and mounting secrets. Fairing also added a config map for a deployer and bug fixes for TensorRTSpec. The workflows enabled by [Kale](https://github.com/kubeflow-kale/kale) include the ability to write model code in a notebook and then automatically build a Kubeflow pipeline that deploys, trains and tunes that model efficiently, using Katib and cached pipeline steps. Kubeflow 1.1 also delivers stable release deliveries of [MXNet and XGBoost operators](https://github.com/kubeflow/common/blob/master/ROADMAP.md), which simplify distributed training on multiple nodes and speeds model creation.
+
+The isolation and security feature deliveries include Private GKE and Anthos support, a stable version of Kubeflow Pipelines with [Multi-User Kubeflow Pipelines](https://github.com/kubeflow/pipelines/projects/1) support, and a process for Kubeflow container image scanning, [CVE reporting](https://docs.google.com/spreadsheets/d/1ijWIyjGQpDy68-vjBmLyFY0U1oGCT--A0nVGgzLGDhU/edit#gid=0), and an optional process for distroless image creation. 1.1 also includes [options for authentication and authorization](https://github.com/kubeflow/kubeflow/issues/4960). This includes the option for administrators to turn off self-service namespace creation mode, as admins may have other processes for namespace creation. The Community also developed a best practice to build user authorization in Kubeflow web apps using subject access review.
+
+The installation and operations of Kubeflow have been enhanced to support GitOps methodologies. Several Kubeflow platform providers and software support vendors are developing time-saving GitOps processes to simplify and codify the installation, configuration and operations of the various layers in the Kubeflow 1.1 hardware and software stack. Some examples are provided in the next section.
+
+**More details and 1.1 tutorials**
+
+Kubeflow 1.1 includes many technical enhancements, which are being delivered via the Community's release process. Details on the application feature development can be found in the [1.1 KanBan Board](https://github.com/orgs/kubeflow/projects/26) and in the [Kubeflow Roadmap](https://github.com/kubeflow/kubeflow/blob/master/ROADMAP.md). As the Kubeflow application improvements are merged, the platform teams (GCP, AWS, IBM, Red Hat, Azure, and Arrikto MiniKF) are working to validate the feature improvements on their respective environments.
```

Comment:
Community -> community

Reply:
I think Community should be capitalized as it refers to a specific community